capture_hdr_loop.py

```diff
@@ -9,60 +7,47 @@
 def _main():
     app = zivid.Application()
 
-    print("Connecting to the camera")
+    print("Connect to the camera")
```

Comment:
When I started writing code samples I was using this tense and I was told that using it to print out stuff may seem as the pc is requesting things from the user, e.g. to connect to a camera or to configure the camera settings, rather than informing the user what is the pc doing atm. We need to check this with SE and figure out what to adopt as Zivid default.

Reply:
Ok

Reply:
Actually, I agree with your comment. We don't have to discuss -ing endings here.